### ISSUE TITLE
Question/Request: Action on release?

### ISSUE THREAD

**Issue by RiverMatt** [kozec/sc-controller#538]
Is there a way to make a keypress happen on release of the button? For example, when the left trigger is soft pressed, shift is pressed and released; then when the left trigger is released, shift is pressed and released again.

Also, is there an API somewhere so I could write my own commands/macros?

**Comment by kozec** [comment]
Yes, but [only as custom command](https://github.com/kozec/sc-controller/wiki/Custom-Action-Examples-and-Explanations#-releasedaction). Stuff described on that wiki page should allow you to do some more complicated tasks, and if everything else fails, you can always use `shell` and run some real script.
